### PR TITLE
Fix for issue #359

### PR DESCRIPTION
**Pull request by mjokic**
Fix for issue #359  Invoke-PSRemoting blocks current agent.
I have added the 'AsJob' parameter which will run the new agent as a background job which won't block the current agent.

### CODE REVIEW

File: lib/modules/powershell/lateral_movement/invoke_psremoting.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def generate(self, obfuscate=False, obfuscationCommand=""):
         if (self.options['AMSIBypass2']['Value']).lower() == 'true':
             AMSIBypass2 = True
 
-        script = """Invoke-Command """
+        script = """Invoke-Command -AsJob"""
```
Comment (Hubbl3) [inline lib/modules/powershell/lateral_movement/invoke_psremoting.py:142]: change back to original

Comment (Hubbl3) [reply lib/modules/powershell/lateral_movement/invoke_psremoting.py:142]: At line 24 change Background from False to true

**Review by Hubbl3 — CHANGES_REQUESTED** [review]

**Comment by mjokic** [comment]
I have applied the changes.

**Comment by mpgn** [comment]
Happy to see my issue was valid ahaha

**Review by Cx01N — APPROVED** [review]

**Review by Hubbl3 — APPROVED** [review]

**Pull request merged**